tools/bin/commands/util/operator_funcs

```diff
@@ -36,3 +36,75 @@ check_operator_requirements() {
 
     echo "OK"
 }
+
+default_binary_path() {
+    local abs_path="$( cd "$( dirname "${BASH_SOURCE[0]}" )/../binaries" >/dev/null 2>&1 && pwd )"
+
+    if [[ -z "$abs_path" ]] ; then
```

Comment:
```suggestion
    if [[ -d "$abs_path" ]] ; then
```

Reply:
it is a matter of taste and clarity ... `-z` catches issues as well as `-d`. Including it in a different PR